Command/EventLogCleanupCommand.php

```diff
@@ -42,13 +42,14 @@ protected function configure(): void
                     new InputOption('dry-run', 'r', InputOption::VALUE_NONE, 'Do a dry run without actually deleting anything.'),
                     new InputOption('campaign-lead', 'c', InputOption::VALUE_NONE, 'Purge only Campaign Lead Event Log Records'),
                     new InputOption('lead', 'l', InputOption::VALUE_NONE, 'Purge only Lead Event Log Records'),
-                    new InputOption('email-stats', 'm', InputOption::VALUE_NONE, 'Purge only Email Stats Records + Email Stats Devices'),
+                    new InputOption('email-stats', 'm', InputOption::VALUE_NONE, 'Purge only Email Stats Records the referenced emails entry is currently not published and purge Email Stats Devices. Important: If referenced email is ever switched back to published, the contacts will get the email again.'),
```

Comment:
Correction: "Purge only Email Stats Records **where** the referenced email~~s~~ entry..."

Reply:
Thanks @moongazer for this spelling recommendation - implemented that with my current commit.